src/runtime_src/core/edge/drm/zocl/scu.c

```diff
@@ -92,11 +92,33 @@ stat_show(struct device *dev, struct device_attribute *attr, char *buf)
 }
 static DEVICE_ATTR_RO(stat);
 
+ssize_t show_sc_pid(u32 sc_pid, char *buf)
+{
+	ssize_t sz = 0;
+	char *fmt = "%u\n";
+
+	sz += scnprintf(buf+sz, PAGE_SIZE - sz, fmt,
+			sc_pid);
+
+	return sz;
+}
+
+static ssize_t
+sc_pid_show(struct device *dev, struct device_attribute *attr, char *buf)
+{
+	struct platform_device *pdev = to_platform_device(dev);
+	struct zocl_scu *scu = platform_get_drvdata(pdev);
+
+	return show_sc_pid(scu->sc_pid, buf);
+}
+static DEVICE_ATTR_RO(sc_pid);
```

Comment:
```suggestion
static DEVICE_ATTR_RO(status);
```
If there is every additional data would we want to create a different node? We could place it all into one attribute file to reduce the amount of files to access. Renaming the attribute to `status` mimics the behavior of the `/proc/` entries. The PID of the process in `/proc/` is also stored in the `status` attribute so the logic matches up. Any additional information we would ever need to add could be easily parsed into the PS kernel I am developing by adding another row of data.